pulsar/internal/lookup_service.go

```diff
@@ -284,12 +291,24 @@ type httpLookupService struct {
 	metrics             *Metrics
 }
 
-func (h *httpLookupService) getBrokerAddress(ld *httpLookupData) (logicalAddress *url.URL,
+func (h *httpLookupService) GetBrokerAddress(brokerServiceURL, brokerServiceURLTls string,
```

Comment:
As I commented, can we pass one of these(brokerServiceURL, brokerServiceURLTls)?

Reply:
Done!